addon/-private/data-view/radar/radar.js

```diff
@@ -36,6 +37,7 @@ export default class Radar {
     this._estimateHeight = 0;
     this._scrollTopOffset = 0;
     this._scrollContainerHeight = 0;
+    this._transformScale = 0;
```

Comment:
shouldn't the default be `1` ?

Reply:
yes, was just defaulting it to a number but `1` would make more sense